src/common.rs

```diff
@@ -5,6 +5,13 @@
 
 #![allow(missing_docs)]
 
+mod sealed {
+    /// A [sealed trait](https://predr.ag/blog/definitive-guide-to-sealed-traits-in-rust/)
+    /// which stops this trait from being derived outside the library. This could be relaxed
+    /// in the future if there is are good reasons to allow outside impls.
```

Comment:
```suggestion
    /// A [sealed trait](https://predr.ag/blog/definitive-guide-to-sealed-traits-in-rust/)
    /// which stops [super::FloatFuncs] from being implemented outside kurbo. This could be relaxed
    /// in the future if there are good reasons to allow outside impls.
```
I'd also probably change 'outside' to 'external' in the final line

Reply:
Should be fixed in [4684dac](https://github.com/linebender/kurbo/pull/332/commits/4684dacad8fd2fb5929a510364c9196ae5c7ec32)

Reply:
I think this link fails with clippy because of the `#[cfg(not(features="std")))]` conditional compilation?